libs/ui/core/src/lib/brain/error-options.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import type { AbstractControl, FormGroupDirective, NgForm } from '@angular/forms';
+
+/** Error state matcher that matches when a control is invalid and dirty. */
+@Injectable()
+export class ShowOnDirtyErrorStateMatcher implements ErrorStateMatcher {
+	isErrorState(control: AbstractControl | null, form: FormGroupDirective | NgForm | null): boolean {
```

Comment:
These names should probably be adjusted to fit the variable name

Reply:
I think i can change it to `isInValid`? any suggestion?